skaffold-operator.yaml

```diff
@@ -247,3 +247,18 @@ deploy:
         replicaCount: 2
       createNamespace: true
       wait: true
+profiles:
+- name: debug
+  activation:
+  - command: debug
+  patches:
+  - op: add
+    path: /deploy/helm/releases/0/setValueTemplates
```

Comment:
Is this JSON patch merging the `setValueTemplates` section or is it replacing the existing values?

Reply:
Good catch, `setValueTemplates` is not merging but replacing the whole `value` section. Thus, I replaced it with `setValues`.
That's weird. What's the difference between `add` and `replace` then 🤔 

/hold cancel